src/promptflow/promptflow/executor/_process_manager.py

```diff
@@ -280,9 +281,13 @@ def ensure_healthy(self):
         # The normal state of the spawned process is 'running'. If the process does not start successfully
         # or exit unexpectedly, its state will be 'zombie'.
         if psutil.Process(self._spawned_fork_process_manager_pid).status() == "zombie":
-            bulk_logger.error("The spawned fork process manager failed to start.")
-            ex = SpawnedForkProcessManagerStartFailure()
-            raise ex
+            with open(".promptflow/spawned_fork_process_manager_stderr.log", "r") as f:
```

Comment:
Better to put the path in a constant rather than copying arround

Reply:
fixed